src/api/app/models/group.rb

```diff
@@ -11,6 +11,9 @@ class Group < ApplicationRecord
   has_many :event_subscriptions, dependent: :destroy, inverse_of: :group
   has_many :reviews, dependent: :nullify, as: :reviewable
 
+  has_many :notifications, dependent: :destroy
+  has_many :rss_feed_items, -> { order(id: :desc) }, class_name: 'Notification::RssFeedItem', dependent: :destroy
```

Comment:
We don't have any created_at field at the notifications table... maybe we should